packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 5.2.0 - Sep 19, 2023
+
+This minor version update of Teams Toolkit addresses your requests with new features and bug fixes, including import exiting SharePoint Framework solution, new app tempaltes for Link Unfurling and AI Chat Bot and much more. We previously shared these incremental changes in the prerelease version and through a series of blog posts:
```

Comment:
"This minor version update of Teams Toolkit includes new features and bug fixes based on your feedback. The new features include the ability to import an existing SharePoint Framework solution, new project templates for Link Unfurling and AI Chat Bot, and more. We previously shared these incremental changes in the prerelease version and through a series of blog posts:"